ref/api-reference.md

```diff
@@ -1,7 +1,174 @@
-The documentation in this section provides a reference for the following APIs provided by Anbox Cloud:
+All communication between AMS and its clients happen using a RESTful API over HTTP. This API is encapsulated over either TLS (for remote operations) or a Unix socket (for local operations).
+
+## APIs and their structure
+Anbox Cloud provides the following APIs:
+
 * AMS HTTP API
 * Anbox Cloud HTTP API
 * Stream Gateway API
 * Anbox Platform SDK API
 
-Also see [SDKs](https://discourse.ubuntu.com/t/anbox-cloud-sdks/17844) for a more detailed description of the SDKs that Anbox Cloud provides.
+All these APIs except for the [Anbox HTTP API](https://discourse.ubuntu.com/t/anbox-http-api/17819) have an auto-generated Swagger specification describing its API endpoints.
```

Comment:
The Anbox Platform SDK API is generated by in [doxygen](https://www.doxygen.nl/) actually

Reply:
I have made the sentence more generic.